tests/unit/LedgerConsentContinuationOut.spec.ts

```diff
@@ -20,7 +20,12 @@ describe('Display ConsentContinuationOut component', () => {
         index: 123,
         filing: {
           documentsLink: 'http://localhost/endpoint',
-          displayName: '6-Month Consent to Continue Out'
+          displayName: '6-Month Consent to Continue Out',
+          data: {
+            consentContinuationOut: {
+              expiry: '2023-08-17T07:59:00.000000+00:00'
```

Comment:
I think this is equivalent to "Aug 16 at 11:59 PM", so the check on line 40 is wrong.

I assume the current code looks at date only, but you now need to convert the whole date-time to get the proper value for display.

Reply:
So we talked about this.

Here, line 26 is actually "Aug 17 at 00:59 AM", but I saw you fix this and you will update the expected value.

Resolving.

Reply:
I tested this again and also looked at what we get back from API. I this this is correct. See the response from API
![image](https://github.com/bcgov/business-filings-ui/assets/116035339/44e332ce-485f-4808-85f8-45d3dfaf89bf)

Reply:
So this test is still wrong. Either use 7:59 for a winter date or 6:59 for a summer date.